infra/monitoring.tf

```diff
@@ -8,6 +8,7 @@ module "monitoring_hudi" {
   database_name   = "hudi"
   table_name      = "hudi_benchmark_ro"
   output_format   = "hudi"
+  module_name     = "hudi"
```

Comment:
This is redundant, read comment in main.tf

Reply:
is redundant in this case as we are comparing hudi vs json but in case we compare cow vs more the output format and module name wont match:

output format: hudi in both cases
module name: hudi-cow / hudi-mor

Reply:
Then do the format string with module name

Reply:
Then please change the module_name to the table format (mor or cow) you are reading in this case, to avoid confussion

Reply:
And maybe the output_format variable can be reused using "mor" or "cow", up to you